File: Api/rest_assured/src/test/java/service/UserService.java
package service;

import io.restassured.response.Response;
import model.User;
import support.enums.ApiPath;
import support.enums.BaseUri;

import java.util.HashMap;
import java.util.Map;

import static support.utils.RestContext.*;

public class UserService {

    static User user;
    public static void createUser(String name, String email, String password, Boolean administrador) {
        initRequest();

        setPath(BaseUri.URLBASE.getPath(), ApiPath.USER.getPath());

        Map<String, String> header = new HashMap<String, String>();
        header.put("Content-type", "application/json");
        setHeader(header);

        Object users = user.builder().nome(name).email(email).password(password).administrador(administrador).build();
        setBody(users.toString());

        postRequest();

    }
}

File: Api/rest_assured_kotlin/src/test/java/service/UserService.kt
package service

import model.User
import support.utils.RestContext
import support.enums.BaseUri
import support.enums.ApiPath
import java.util.HashMap

object UserService {
    @JvmStatic
    fun createUser(name: String?, email: String?, password: String?, administrador: Boolean?) {
        println(name)
        RestContext.initRequest()
        RestContext.setPath(BaseUri.URLBASE.path, ApiPath.USER.path)
        val header: MutableMap<String, String> = HashMap()
        header["Content-type"] = "application/json"
        RestContext.setHeader(header)
        val user = User(name,email,password,administrador)
        RestContext.setBody(user.toString())
        RestContext.postRequest()
    }
}